src/app/public/modules/dropdown/dropdown.component.ts

```diff
@@ -303,19 +469,25 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
     this.messageStream.next({ type });
   }
 
-  private positionPopover(): void {
-    this.popover.positionNextTo(
-      this.triggerButton,
-      'below',
-      this.alignment
-    );
-  }
+  private positionDropdownMenu(): void {
+    this.isVisible = false;
+    this.createOverlay();
+    this.changeDetector.markForCheck();
+
+    this.windowRef.getWindow().setTimeout(() => {
```

Comment:
This was kept in because it was in the previous implementation (trying to avoid needless changes), but if you don't mind, I'll remove it; I'm not a fan of using `windowRef` if it can be avoided.

Reply:
Me too. Yeah, go for it.